config/initializers/elasticsearch.rb

```diff
@@ -1,9 +1,37 @@
-config = {
-  host: ENV.fetch("ES_HOST", "http://127.0.0.1:9200"),
-  transport_options: {
-    request: { timeout: 5 }
+require 'base64'
+
+vcap_services = ENV.fetch("VCAP_SERVICES", nil)
+
+if vcap_services
```

Comment:
I'd suggest using the stricter `vcap_services.present?`. That protects us against the case when `vcap_services` is an empty string.

```
if ""
  "foo"
else
  "bar"
end

=> "foo"
```

```
"".present?
=> false
```